demo/nodejs/example.js

```diff
@@ -2,9 +2,9 @@ const ex = require('../../ffi/nodejs/build/Release/rgb_node')
 
 const config = {
     network: "testnet",
-    stash_endpoint: "ipc:/tmp/rgb-node/testnet/stashd.rpc",
+    stash_endpoint: "lnpz:/tmp/rgb-node/testnet/stashd.rpc",
     contract_endpoints: {
-        Fungible: "ipc:/tmp/rgb-node/testnet/fungibled.rpc"
+        Fungible: "lnpz:/tmp/rgb-node/testnet/fungibled.rpc"
```

Comment:
Hm, why do we use uppercase letter? Does it work?

Reply:
We use that because [it was already like that](https://github.com/LNP-BP/rgb-node/blob/v0.1.0-rc.1/ffi/nodejs/example.js#L7), so I assumed it was ok. But considering there was a lot of stuff broken even before moving the ffi here, we can consider it probably wrong. I'll test it without capitalization and then let you know

Reply:
using `fungible` instead of `Fungible` causes this error:
```
Json(Error("unknown variant `fungible`, expected one of `Fungible`, `Collectible`, `Identity`", line: 1, column: 11))
```